website/content/en/preview/getting-started/_index.md

```diff
@@ -82,8 +82,9 @@ You can add the details of the newly created EKS cluster to your local kubectl a
 ```bash
 aws eks update-kubeconfig --region ${AWS_DEFAULT_REGION} --name ${CLUSTER_NAME}
```

Comment:
I'm also not sure why this was added. eksctl will update your kubeconfig as part of creating the cluster

Reply:
removing